docs/reference/tab-widgets/inference-api/infer-api-task.asciidoc

```diff
@@ -0,0 +1,55 @@
+// tag::cohere[]
+
+[source,console]
+------------------------------------------------------------
+PUT _inference/text_embedding/cohere_embeddings <1>
+{
+    "service": "cohere",
+    "service_settings": {
+        "api_key": "<api_key>", <2>
+        "model": "embed-english-light-v3.0" <3>
+    },
```

Comment:
How about we also add the field `embedding_type` to the example as well. Just showing that users can set it to `float` or `int8`.